components/FLoader.vue

```diff
@@ -0,0 +1,45 @@
+<template lang="pug">
+FLogo.FLoader(
+  variant="image-only"
+  :height="height"
+  :width="width"
+  :color="color"
+)
+</template>
+
+<style lang="stylus">
+.FLoader
+  animation 1800ms rotate infinite forwards cubic-bezier(0.6, 0.52, 0.3, 0.85)
```

Comment:
Je mettrais ça pour correspondre plus à l'anim de Marco
- vidéo : 13 secondes => 5 animations => 1 anim fait 2.6sec
- accélération plus forte, easing plus long

```suggestion
  animation 2600ms rotate infinite forwards cubic-bezier(.17, 0, .2, 1)
```

Teste et dis-moi ce que t'en penses